scripts/test_wheel.sh

```diff
@@ -39,6 +42,7 @@ while [ $# -gt 0 ]
     --docker-python3.5)     USE_DOCKER=1;DOCKER_PYTHON=3.5;;
     --docker-python3.6)     USE_DOCKER=1;DOCKER_PYTHON=3.6;;
     --docker-python3.7)     USE_DOCKER=1;DOCKER_PYTHON=3.7;;
+    --minimal-build)        USE_MINIMAL=1;;
```

Comment:
Let's just keep this `--minimal` to be consistent between this script and the setup.py script.

Reply:
done.